core-bundle/src/Util/SimpleTokenParser.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Util;
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\LogLevel;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+
+class SimpleTokenParser implements LoggerAwareInterface
+{
+    use LoggerAwareTrait;
+
+    /**
+     * @var ExpressionLanguage
+     */
+    private $expressionLanguage;
+
+    public function __construct(ExpressionLanguage $expressionLanguage)
+    {
+        $this->expressionLanguage = $expressionLanguage;
+    }
+
+    /**
+     * Parse simple tokens.
+     *
+     * @param string $subject The string to be parsed
```

Comment:
There‘s no need for theses docblocks, as the argument name and type are already given by PHP7

Reply:
I know, but what's with the ones adding descriptions? Would we drop the typehint only (`@param $subject The string to be parsed`)? And if so, could the CS fixer do this? :smile:

Reply:
`The string to be parsed` is irrelevant anyway. It's obvious.